prooph/event-store-symfony-bundle/0.4/config/packages/prooph_event_store.yaml

```diff
@@ -0,0 +1,14 @@
+prooph_event_store:
+    stores:
+        default:
+            event_store: 'app.event_store.default'
+            # repositories:
```

Comment:
Are these comments really needed? 
We usually recommend to put this in documentation instead.